src/main/java/com/taskadapter/redmineapi/internal/Transport.java

```diff
@@ -259,7 +261,7 @@ public User getCurrentUser(NameValuePair... params) throws RedmineException {
 	/*
 	 * note: This method cannot return the updated object from Redmine because
 	 * the server does not provide any XML in response.
-	 * 
```

Comment:
there are multiple formatting changes like this one. they make review more complex and git history less clean...